assets/scss/6-components/story-block/story-block-alt.html

```diff
@@ -1,9 +1,9 @@
 <article class="c-story-block c-story-block--stack-from-bp-m">
-  <figure class="l-display-block l-width-full">
-    <a href="#"><img class="l-display-block l-width-full" src="/img/DEFAULT_LEAD_ART.jpg" alt="Story block alt tag"></a>
-  </figure>
-  <section>
+  <div class="c-story-block__text">
     <!-- headline -->
-    <h3 class="t-size-s t-serif t-lh-s t-lsp-s t-links-underlined-hover has-xxxs-btm-marg js-balance"><a class="has-text-black-off"><a href="#">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Facilis ea fuga ipsum doloremque porro voluptatem fugit.</a></h3>
-  </section>
-</article>
\ No newline at end of file
+    <h3 class="t-size-s t-serif t-lh-s t-lsp-s t-links-underlined-hover has-xxxs-btm-marg"><a class="has-text-black-off"><a href="#">The best and worst moments in Texas politics of 2019</a></h3>
```

Comment:
I imagine this could probably be an `h2` or an `h3`, depending on the page and context, right?

Reply:
sure could. I can add that in a comment if it helps